Libraries/LibWeb/Selection/Selection.cpp

```diff
@@ -134,8 +134,11 @@ String Selection::direction() const
 // https://w3c.github.io/selection-api/#dom-selection-getrangeat
 WebIDL::ExceptionOr<GC::Ptr<DOM::Range>> Selection::get_range_at(unsigned index)
 {
-    // The method must throw an IndexSizeError exception if index is not 0, or if this is empty.
-    if (index != 0 || is_empty())
+    // The method must throw an IndexSizeError exception if index is not 0, or if this is empty or either focus or anchor is not in the document tree.
+    auto is_focus_in_document_tree = focus_node() && focus_node()->in_a_document_tree();
+    auto is_anchor_in_document_tree = anchor_node() && anchor_node()->in_a_document_tree();
```

Comment:
The spec states "in _the_ document tree", not "in _a_ document tree". I think this is trying to confirm that both the anchor and focus are in the same document as the selection itself.

Reply:
I guess I resolved that